src/Config/FileConfigManager.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+namespace Spatie\Ignition\Config;
+
+use Spatie\Ignition\Contracts\ConfigManager;
+
+class FileConfigManager implements ConfigManager
+{
+    private const SETTINGS_FILE_NAME = '.ignition.json';
```

Comment:
Maybe we can rename this to `CONFIG_FILE_NAME` to keep it in line with the `ConfigManager`, etc...

```suggestion
    private const CONFIG_FILE_NAME = '.ignition.json';
```

Reply:
Hello,

I kind of agree, but to me, the `config/ignition.php` is a config file and `.ignition.json` is a settings file. One of the reasons for this separation is the Laravel Ignition button naming (we have a Save settings button there). So, not to confuse users I tried to differentiate these two concepts by introducing this term `settings file`. That's why I would like to stick to this name `settings file` and use it for this file if you don't mind.

However, if you don't like this differentiation at all let's change it. @AlexVanderbist please let me know about your decision.